microsoft-edge/webview2/release-notes/index.md

```diff
@@ -63,6 +63,305 @@ No additional APIs have been promoted to Stable and added in this Release SDK.
 <!-- end of Jan 2025 Release SDK -->
 
 
+<!-- ====================================================================== -->
+## 1.0.3079-prerelease
+
+Release Date: January 23, 2025
+
+[NuGet package for WebView2 SDK 1.0.3079-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3079-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 134.0.3079.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+The following Experimental APIs have been added in this Prerelease SDK.
+
+
+<!-- ---------- -->
+###### DefaultBackgroundColor API<!-- todo: feature-oriented heading -->
+
+<!-- todo: lead-in - https://github.com/MicrosoftEdge/WebView2Feedback/pull/4981/files -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2ControllerOptions` Class:
+   * [CoreWebView2ControllerOptions.DefaultBackgroundColor Property](/dotnet/api/microsoft.web.webview2.core.corewebview2controlleroptions.defaultbackgroundcolor?view=webview2-dotnet-1.0.3079-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ControllerOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controlleroptions?view=webview2-winrt-1.0.3079-prerelease&preserve-view=true)
+   * [CoreWebView2ControllerOptions.DefaultBackgroundColor Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controlleroptions?view=webview2-winrt-1.0.3079-prerelease&preserve-view=true#defaultbackgroundcolor)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalControllerOptions3](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcontrolleroptions3?view=webview2-1.0.3079-prerelease&preserve-view=true)
+  * [ICoreWebView2ExperimentalControllerOptions3::get_DefaultBackgroundColor](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcontrolleroptions3?view=webview2-1.0.3079-prerelease&preserve-view=true#get_defaultbackgroundcolor)
+  * [ICoreWebView2ExperimentalControllerOptions3::put_DefaultBackgroundColor](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcontrolleroptions3?view=webview2-1.0.3079-prerelease&preserve-view=true#put_defaultbackgroundcolor)
+
+---
+
+
+<!-- ---------- -->
+###### FrameCreatedEvent API<!-- todo: feature-oriented heading -->
+
+<!-- todo: lead-in - https://github.com/MicrosoftEdge/WebView2Feedback/pull/4982 -->
```

Comment:
add lead-in for F.C.E., see https://github.com/MicrosoftEdge/WebView2Feedback/pull/4982